.k8/helm3/ssv-liquidator/templates/ssv_liquidator_worker.yml

```diff
@@ -85,15 +85,15 @@ spec:
               name: config-secrets
               key: ssv_sync
 
-#        volumeMounts:
-#        - mountPath: "/data/"
-#          name: ssv-liquidator-worker
+        volumeMounts:
+        - mountPath: "/data/"
+          name: ssv-liquidator-worker-tmp
         resources:
           {{- toYaml .Values.resources | nindent 12 }}
-#      volumes:
-#      - persistentVolumeClaim:
-#          claimName: ssv-liquidator-worker
-#        name: ssv-liquidator-worker
+      volumes:
+      - persistentVolumeClaim:
+          claimName: ssv-liquidator-worker-tmp
```

Comment:
Better to use worker number instead of `tmp`, otherwise all workers will share the same volume